ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -580,15 +580,15 @@ private class JdbcLedgerDao(
         ledgerEntries.foreach { case (offset, entry) =>
           entry match {
             case tx: LedgerEntry.Transaction =>
-              val submitterInfo =
-                for (
-                  appId <- tx.applicationId;
-                  actAs <- if (tx.actAs.isEmpty) None else Some(tx.actAs);
+              val completionInfo =
+                for {
+                  appId <- tx.applicationId
+                  actAs <- if (tx.actAs.isEmpty) None else Some(tx.actAs)
                   cmdId <- tx.commandId
-                )
-                  yield state.SubmitterInfo(actAs, appId, cmdId, Instant.EPOCH)
+                  subId <- tx.submissionId
+                } yield state.CompletionInfo(actAs, appId, cmdId, None, subId)
```

Comment:
Could you expand the param names here and in similar occurrences?

Reply:
I'm trying not to touch `JdbcLedgerDao` more than necessary because of inevitable conflicts due to its size. That said, I hit these lines anyway, so why not?